FirebasePerformance/Sources/AppActivity/FPRTraceBackgroundActivityTracker.m

```diff
@@ -15,7 +15,9 @@
 #import "FirebasePerformance/Sources/AppActivity/FPRTraceBackgroundActivityTracker.h"
 
 #import <Foundation/Foundation.h>
+#if !TARGET_OS_OSX
 #import <UIKit/UIKit.h>
```

Comment:
The import might not be needed at all if this is the only change.

Reply:
Got it. Done.